src/main/java/mate/academy/mybookshop/entity/RoleEntity.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.mybookshop.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+import org.springframework.security.core.GrantedAuthority;
+
+@Data
+@Entity
+@Table(name = "roles")
+@SQLDelete(sql = "UPDATE roles SET is_deleted = true WHERE id = ?")
+@SQLRestriction(value = "is_deleted=false")
+public class RoleEntity implements GrantedAuthority {
```

Comment:
I prefer naming like that, but it is up to you
```suggestion
public class Role implements GrantedAuthority {
```

Reply:
for me better RoleEntity